app/models.py

```diff
@@ -876,6 +876,9 @@ def check_code(self, cde):
 
 NOTIFICATION_STATUS_LETTER_ACCEPTED = 'accepted'
 NOTIFICATION_STATUS_LETTER_ACCEPTED_PRETTY = 'Accepted'
+NOTIFICATION_STATUS_LETTER_RECEIVED = 'received'
```

Comment:
Should received by capitalised? Or is this defined by DVLA?

Reply:
Don't think we use capitalisation internally for the different statuses